Document/0x04e-Testing-Authentication-and-Session-Management.md

```diff
@@ -118,7 +118,7 @@ Security experts used to recommend using session-based authentication and mainta
 
 To prevent tampering cryptographic signatures are added to client-side tokens. Of course, things may go wrong, and popular implementations of stateless authentication have been vulnerable to attacks. For example, the signature verification of some JSON Web Token (JWT) implementations could be deactivated by [setting the signature type to "None."](https://auth0.com/blog/critical-vulnerabilities-in-json-web-token-libraries/) We'll discuss this attack in more detail in the "Testing JSON Web Tokens" chapter.
 
-#### Best Practices for Passwords
+#### Best Practices for Passwords (MSTG‑AUTH‑5)
```

Comment:
There is quite a lot about passwords, shouldn't we better have a "### Testing Password Policies" section? Consider adding:

- Best Practices for Passwords (MSTG‑AUTH‑5) + sub-sections
- Running a Password Dictionary Attack (MSTG‑AUTH‑6)

Reply:
Makes sense. I merged now everything into one password test case (line 89 onwards)